development/custom-image/install-deps-debian.sh

```diff
@@ -102,6 +102,19 @@ sudo apt-cache madison google-fluentd
 sudo apt-get install -y 'google-fluentd=1.*'
 sudo apt-get install -y google-fluentd-catch-all-config
 
+# install go
+ export GO_VERSION=1.18.2
```

Comment:
Can you put `GO_VERSION` at the top with the rest of dependencies versions?

Reply:
GO_VERSION moved